### PR TITLE
Merge develop to master

### PR DESCRIPTION
**Pull request by alalonde**

### CODE REVIEW

File: hasura/migrations/1624277873142_insert_metacartel_ventures_data/up.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  join_button_url, moloch_address, 
+  website_url, guildname, discord_id,
+  status, discord_metadata,
+  discord_invite_url, twitter_url,
+  github_url
+) VALUES
+(
+  'FUNDING', 'MetaCartel Ventures', 
+ 'https://images.squarespace-cdn.com/content/5c8ceb5e2727be114994c1f3/1582727237261-7PIWNXCFPCSVWO0D3DDF/TEH+OG.png?content-type=image%2Fpng', 
+ 'MetaCartel Ventures (Venture DAO) is a for-profit DAO created by the MetaCartel community for the purposes of making investments into early-stage Decentralized Applications (DApps)',
```
Comment (dysbulic) [inline hasura/migrations/1624277873142_insert_metacartel_ventures_data/up.sql:12]: This should en
```suggestion
 'MetaCartel Ventures (Venture DAO) is a for-profit DAO created by the MetaCartel community for the purposes of making investments into early-stage Decentralized Applications (DApps).',
```

File: packages/discord-bot/src/discord/commands/setEthAddress.ts [hunk 2]
```diff
@@ -47,14 +52,21 @@ export abstract class SetEthAddress {
...
     const account = manager.ledger.account(baseIdentityId);
 
-    const existing = account.identity.aliases.find((alias) => {
+    const existingEthAliases = account.identity.aliases.filter((alias) => {
       const parts = sc.core.graph.NodeAddress.toParts(alias.address);
       return parts.indexOf('ethereum') > 0;
     });
 
-    if (existing) {
+    const latestEthAlias = existingEthAliases[existingEthAliases.length - 1];
```
Comment (dysbulic) [inline packages/discord-bot/src/discord/commands/setEthAddress.ts:60]:
```suggestion
    const [latestEthAlias] = existingEthAliases.slice(-1);
```

**Review by dysbulic — APPROVED** [review]
I noticed a missing period and a place where the line could be significantly shortened.

File: packages/discord-bot/src/discord/commands/setEthAddress.ts (hunk 2 above)
Comment (alalonde) [reply packages/discord-bot/src/discord/commands/setEthAddress.ts:60]: More concise, yes, more readable, probably not.

**Pull request merged**